### ISSUE TITLE
Discrete fundus hypopigmentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11393]
ERN EYE suggestion

**Preferred term label:**
Discrete fundus hypopigmentation
Def: Well-demarcated areas of the fundus, with relative reduction in pigment, with respect to the rest of the fundus.
Comment: This is seen in vitiligo, lacunae of CHRPE and achromatic retinal patches. 

**Parent term (use hpo.jax.org/app)**
Parent: Hypopigmentation of the fundus (HP:0007894)

Issue -- the new term would have just one child, Achromatic retinal patches -- but it would only make sense to make a new term for Discrete fundus hypopigmentation if there are multiple child terms
